pontoon/base/static/js/translate.js

```diff
@@ -2992,22 +2992,42 @@ var Pontoon = (function (my) {
       var self = this,
           $forAuthors = $('#filter').find('.for-authors').toggle(authors.length > 0);
 
+      var selectedAuthors = $('#filter .menu li.author.selected').map(function() {
+                                return $.trim($(this).data("type"));
+                            }).get();
+
       $('#filter .menu li.author').remove();
 
       $.each(authors, function() {
-        $forAuthors.after('<li class="author" data-type="' + this.email + '">' +
-          '<figure>' +
-            '<span class="sel">' +
-              '<span class="status fa"></span>' +
-              '<img class="rounded" src="' + this.gravatar_url + '">' +
-            '</span>' +
-            '<figcaption>' +
-              '<p class="name">' + this.display_name + '</p>' +
-              '<p class="role">' + this.role + '</p>' +
-            '</figcaption>' +
-            '<span class="count">' + self.numberWithCommas(this.translation_count) + '</span>' +
-          '</figure>' +
-        '</li>');
+        if (selectedAuthors.includes(this.email)) {
+          $forAuthors.after('<li class="author selected" data-type="' + this.email + '">' +
+            '<figure>' +
+              '<span class="sel">' +
+                '<span class="status fa"></span>' +
+                '<img class="rounded" src="' + this.gravatar_url + '">' +
+              '</span>' +
+              '<figcaption>' +
+                '<p class="name">' + this.display_name + '</p>' +
+                '<p class="role">' + this.role + '</p>' +
+              '</figcaption>' +
+              '<span class="count">' + self.numberWithCommas(this.translation_count) + '</span>' +
+            '</figure>' +
+          '</li>');
+        } else {
+          $forAuthors.after('<li class="author" data-type="' + this.email + '">' +
+            '<figure>' +
+              '<span class="sel">' +
+                '<span class="status fa"></span>' +
+                '<img class="rounded" src="' + this.gravatar_url + '">' +
+              '</span>' +
+              '<figcaption>' +
+                '<p class="name">' + this.display_name + '</p>' +
+                '<p class="role">' + this.role + '</p>' +
+              '</figcaption>' +
+              '<span class="count">' + self.numberWithCommas(this.translation_count) + '</span>' +
+            '</figure>' +
+          '</li>');
+        }
```

Comment:
This is an anti-pattern. You duplicated the entire block, so in case we need to make a change, we need to make it in two places. That's rarelly a good idea.

Instead, you can keep the original block (the entire thing marked in red) and replace the first line with:
```js
var selected = (selectedAuthors.includes(this.email)) ? ' selected' : '';
$forAuthors.after('<li class="author' + selected + '" data-type="' + this.email + '">' +
```